flake.nix

```diff
@@ -3,13 +3,15 @@
 
   inputs = {
     flake-parts.url = "github:hercules-ci/flake-parts";
-    nixpkgs.url = "github:NixOS/nixpkgs/nixos-unstable";
+    nixpkgs.url = "github:NixOS/nixpkgs/master";
```

Comment:
add a comment to document when this can be reverted : https://nixpk.gs/pr-tracker.html?pr=391300